app/root/header/Header.tsx

```diff
@@ -66,10 +68,33 @@ function NavDropDownButton({
 }
 
 export function Header() {
+  const isMobileWidth = useWindowSize().width < 1024
   const email = useEmail()
   const idp = useUserIdp()
   const [expanded, setExpanded] = useState(false)
   const [userMenuIsOpen, setUserMenuIsOpen] = useState(false)
+  const menuRef = useRef<HTMLDivElement>(null)
+  const userMenuRef = useRef<HTMLDivElement>(null)
+
+  const [mobile, setMobile] = useState(isMobileWidth ?? false)
+
+  useEffect(() => {
+    setMobile(isMobileWidth)
+  }, [isMobileWidth])
```

Comment:
This is probably unnecessary. The point of `useWindowSize()` is so that you can have props that depend functionally rather than imperatively on the window size.

Reply:
Yep, removed in cb419f1